pkg/serverless/daemon/interval_test.go

```diff
@@ -86,11 +86,9 @@ func TestInvocationInterval(t *testing.T) {
 	}
 
 	for i := 0; i < 19; i++ {
-		time.Sleep(100 * time.Millisecond)
 		d.lastInvocations = append(d.lastInvocations, time.Now())
 		assert.Equal(time.Duration(0), d.InvocationInterval(), "we should not compute any interval just yet since we don't have enough data")
 	}
-	time.Sleep(100 * time.Millisecond)
```

Comment:
The specific timestamps set in the `d.lastInvocations` slice do not actually matter.